docker/Dockerfile

```
@@ -112,6 +111,33 @@ ENV CC /opt/conda/bin/x86_64-conda_cos6-linux-gnu-gcc
 ENV CXX /opt/conda/bin/x86_64-conda_cos6-linux-gnu-g++
 ENV CPP /opt/conda/bin/x86_64-conda_cos6-linux-gnu-cpp
 
+FROM apt-deps as apt-coverage-deps
+RUN apt install -y lcov curl
+ENV GCOV /usr/bin/gcov
+
+FROM conda-deps as conda-coverage-deps
+RUN mamba install -y lcov curl
+ENV GCOV /opt/conda/bin/x86_64-conda-linux-gnu-gcov
+
+FROM ${pkg_mgr}-coverage-deps as cyclus-coverage
+ARG make_cores=2
+
+COPY . /cyclus
+WORKDIR /cyclus
+
+RUN python install.py -j 2 --build-type=Release --core-version 999999.999999 --allow-milps --code_coverage
```

Is there a reason not to parallelize this further?
```suggestion
RUN python install.py -j ${make_cores} --build-type=Release --core-version 999999.999999 --allow-milps --code_coverage
```

---

Nope just oversight on my part.  Updated Dockerfile accordingly